prescient/stats/hourly_stats.py

```diff
@@ -138,121 +142,93 @@ def __init__(self, options, day: date, hour: int):
-
-        self.renewable_gen_cleared_DA = { g : ruc_market.renewable_gen_cleared_DA[g,time_index] \
-                                          for g in extractor.get_nondispatchable_generators(sced) }
-
-        self.thermal_reserve_cleared_DA = { g : ruc_market.thermal_reserve_cleared_DA[g,time_index] \
-                                            for g in extractor.get_thermal_generators(sced) }
-
-        self.thermal_gen_revenue = dict()
-        self.renewable_gen_revenue = dict()
-        for b in extractor.get_buses(sced):
-            price_DA = self.planning_energy_prices[b]
-            price_RT = self.observed_bus_LMPs[b]
-
-            for g in extractor.get_thermal_generators_at_bus(sced, b):
-                self.thermal_gen_revenue[g] = \
-                    self.thermal_gen_cleared_DA[g]*price_DA + \
-                    (self.observed_thermal_dispatch_levels[g] - self.thermal_gen_cleared_DA[g])*price_RT
-
-            for g in extractor.get_nondispatchable_generators_at_bus(sced, b):
-                self.renewable_gen_revenue[g] = \
-                    self.renewable_gen_cleared_DA[g]*price_DA + \
-                    (self.observed_renewables_levels[g] - self.renewable_gen_cleared_DA[g])*price_RT
-
-
-
-        r_price_DA = self.planning_reserve_price
-        r_price_RT = self.reserve_RT_price
-        self.thermal_reserve_revenue = { g : self.thermal_reserve_cleared_DA[g]*r_price_DA + \
-                                                ( self.observed_thermal_headroom_levels[g] - \
-                                                   self.thermal_reserve_cleared_DA[g] )*r_price_RT
-                                         for g in extractor.get_thermal_generators(sced) }
-
-        ## TODO: calculate uplift for the day
-        self.thermal_uplift = { g : 0. for g in extractor.get_thermal_generators(sced) }
-
-        ## TODO: calculate uplift for the day
-        self.renewable_uplift = { g : 0. for g in extractor.get_nondispatchable_generators(sced) }
-
+        self.thermal_fleet_capacity = ops_stats.thermal_fleet_capacity
+
+        # Add scalar fields
+        summing_fields = [
+            'sced_runtime',
+            'total_demand',
+            'fixed_costs',
+            'variable_costs',
+            'power_generated',
+            'load_shedding',
+            'over_generation',
+            'reserve_shortfall',
+            'available_reserve',
+            'available_quickstart',
+            'renewables_available',
+            'renewables_used',
+            'renewables_curtailment',
+            'on_offs',
+            'sum_on_off_ramps',
+            'sum_nominal_ramps',
+            'quick_start_additional_costs',
+            'quick_start_additional_power_generated',
+            'reserve_requirement',
+            'price', # ?
+            'reserve_RT_price', # ?
+            'planning_reserve_price' # ?
+           ]
```

Comment:
Some of these fields should be averaged as opposed to summed.

Reply:
Averaged or summed after multiplying by `sced_time_period_length_minutes/60.` (should be the same operation):
- sced_runtime
- total_demand
- power_generated
- load_shedding
- over_generation
- reserve_shortfall
- available_reserve
- available_quickstart
- renewables_available
- renewables_used
- renewables_curtailment
- quick_start_additional_power_generated
- reserve_requirement
- price
- reserve_RT_price
- planning_reserve_price

Summed as is:
- fixed_costs
- variable_costs
- on_offs
- sum_on_off_ramps
- sum_nominal_ramps
- quick_start_additional_costs

I think these are a disjoint covering of the items above...hopefully the units make sense.